src/widgets/search-box/search-box.js

```diff
@@ -68,7 +68,10 @@ const renderer = ({
         refine(getValue(e), false);
       });
       addListener(input, 'keyup', e => {
-        if (e.keyCode === KEY_ENTER) refine(getValue(e));
+        if (e.keyCode === KEY_ENTER) {
```

Comment:
since we have a form now, this can be done in a submit event on the form instead, and the ENTER code can be removed :)

Reply:
Yes indeed :)